src/views/TreeRegistration.vue

```diff
@@ -147,26 +164,60 @@ import axios from "axios"
         month_items: ['January', 'February', 'March', 'April', 'May', 'June', 'July',
                       'August', 'September', 'October', 'November', 'December'],
         propertyCard: '',
-
       }
     },
+    
+    validations: {
+      tree_type: { required },
+      number_of_tree: { required },
+      height_fruit: { required },
+      months: { required },
+      haverst_for_year: { required },
+    },
+    
     computed: {
       icon(){
       return "chevron-right"
       },
-
-      formIsValid () {
-        return this.tree_type !== '' &&
-              this.number_of_tree !== '' &&
-              this.height_fruit !== '' &&
-              this.months !== '' &&
-              this.propertyCard !== ''
-      }
+      tree_type_errors () {
+        const errors = []
+        if (!this.$v.tree_type.$dirty) return errors
+        !this.$v.tree_type.required && errors
+          .push('Tree type cant be black!')
```

Comment:
Essa mensagem, que se repete em outros campos desse formulário ("X cant be black") não faz muito sentido, o que faria mais sentido é ("X must be filled")

Reply:
Alterado. vai no próximo commit de ajustes.

Reply:
A mensagem correta era para ser "X cant be blank" - "não pode ficar em branco". Foi erro de digitação. Alterado para "must be filled".